tests/riscv64/efficientnet_lite0_quant_rvv_test.run

```diff
@@ -0,0 +1,4 @@
+// RUN: ${TEST_MODULE_CMD} --module_file="${BUILD_RISCV_DIR}/efficientnet_lite0_int8_2.tflite.rvv.vmfb" \
+// RUN:   --entry_function=main --function_input="1x224x224x3xui8=0" > %t
+// RUN: ${HOST_TEST_MODULE_CMD} --module_file="${BUILD_RISCV_DIR}/efficientnet_lite0_int8_2.tflite.x86_64.vmfb" \
+// RUN:   --entry_function=main --function_input="1x224x224x3xui8=0" | diff %t -
```

Comment:
While this test using `diff` is concise to write, I'm skeptical about the error messages and general integration with the rest of the project. We should use the check framework instead. See https://github.com/iree-org/iree/blob/main/docs/developers/developing_iree/testing_guide.md#writing-a-test-2

Reply:
I agree if possible, we should use the `check.expect_almost_eq_const` to perform the check. However, in this case, if we really want to test these models in `tests/e2e` framework, we need to check in the mlir file and add the check at the end, which could be huge (deeplab mlir is a 10MB text file). I am not sure it is feasible in this case. :(

Reply:
Within the smoketests for one platform is not the place to be making those decisions. If an integration test for those programs is useful on one platform, it will likely be useful on other platforms too, and then we would need to solve that problem. These tests are already fetching remote files, we could do the same for generic integration tests, even using the check framework.

Reply:
I can extend the `iree_bytecode_module_for_iree_check_test_and_friends` to support rv64 target, however, it would still need another wrapper (in this case, the lit .run file) to add the emulator.

I will see how I can add these into `benchmarks/TFLite/linux-riscv.cmake`, in which we fetch these models already. However, putting them within `benchmarks` still feels awkward.

Reply:
Add a simply python tool to compare the `iree-run-module` stdout between two executions. In this PR, it is used to compare between x86_64 and rv64 RVV; however, it can easily support other cases, cross-compile or not.